src/modules/launcher/Wox.Infrastructure/Wox.cs

```diff
@@ -7,12 +7,12 @@ namespace Wox.Infrastructure
 {
     public static class Constant
     {
-        public const string Wox = "PowerLauncher";
+        public const string ExeFileName = "PowerLauncher";
```

Comment:
No action here for now, but for other peoples benefit what we discussed is renaming all powertoys executables to Power.<Branded_PowerToy_Name>.exe